### PR TITLE
Ahmed

### PR DESCRIPTION
**Pull request by ahmedkilany01**

### CODE REVIEW

File: src/components/pages/ProductDetailsPage/sections/ContentMain/index.jsx [hunk 1]
```diff
@@ -0,0 +1,182 @@
...
+            </div>
+            <BsDot className="icon" />
+            <div>
+              <BsBasket2 className="icon" />
+              <span>32 reviews</span>
+            </div>
+          </StyledRatingContainer>
+          <StyledPricing>
+            <StyledPrice className="first">
+              <Typography as="p" className="selected">
```
Comment (se-bayan-elian) [inline src/components/pages/ProductDetailsPage/sections/ContentMain/index.jsx:77]: u make code more reusable as possible , try to not repeat the styled pricing more than one ,use mock data and map on them

File: src/components/pages/ProductDetailsPage/sections/ContentMain/index.jsx
Comment (se-bayan-elian) [inline src/components/pages/ProductDetailsPage/sections/ContentMain/index.jsx:1]: review all you code not just the this component , HARAM TO BE THIS NUMBER OF LINES

File: src/components/pages/ProductDetailsPage/sections/Description/index.jsx [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+              </div>
+            </Item>
+            <Item>
+              <Image src="/assets/images/cloth/01.png" width={100} height={100} />
+              <div>
+                <Typography as="p">Apple Watch Series Space Gray</Typography>
+                <span>$7.00 - $99.50</span>
+              </div>
+            </Item>
+          </ItemsContainer>
```
Comment (se-bayan-elian) [inline src/components/pages/ProductDetailsPage/sections/Description/index.jsx:95]: also u repeat yourself , us mock data

File: src/components/pages/ProductDetailsPage/sections/ContentMain/index.jsx [hunk 3]
```diff
@@ -0,0 +1,149 @@
...
+              </div>
+              <div>
+                <MdOutlineVerifiedUser className="icon" />
+                <p>Verified Seller</p>
+              </div>
+              <div>
+                <TbWorld className="icon" />
+                <p>Worldwide shipping</p>
+              </div>
+              <button>Send inquiry</button>
```
Comment (mo7mad4) [inline src/components/pages/ProductDetailsPage/sections/ContentMain/index.jsx:128]: Hi Ahmad
This's button is puer , but We are make button component
i solve it for this code 
```js
 <Button variant='primary' className='primary__button'>Send inquiry</Button>
 <Button variant='secondary'>Seller's profile</Button>
 ```
 you make import for this button component and make padding = 5px i think

**Pull request closed without merging**